Userland/Games/Solitaire/Game.cpp

```diff
@@ -113,6 +113,8 @@ void Game::setup(Mode mode)
     m_passes_left_before_punishment = recycle_rules().passes_allowed_before_punishment;
     m_score = 0;
     update_score(0);
+    if (on_undo_available)
+        on_undo_available(false);
```

Comment:
Based on the name I wouldn't expect this to be called when an undo action is *not* available - this really is something like `on_undo_availability_change`, right?

Reply:
Absolutely, `on_undo_availability_change` is such a better name, updated..